Libraries/StyleSheet/__tests__/processTransform-test.js

```diff
@@ -18,13 +18,20 @@ describe('processTransform', () => {
       processTransform([]);
     });
 
+    it('should accept an empty string', () => {
+      processTransform('');
+    });
+
     it('should accept a simple valid transform', () => {
       processTransform([
         {scale: 0.5},
         {translateX: 10},
         {translateY: 20},
         {rotate: '10deg'},
       ]);
+      processTransform(
+        'scale(0.5) translateX(10) translateY(20) rotate(10deg)',
```

Comment:
Translate function values must have units unless they are value `0`

Reply:
@necolas I understand that on web we need to provide units but on react native all dimensions are unitless, and represent density-independent pixels. Which sort of units should we accept in this sting format? If we accept `px` should we convert that to `dp` when passing it down to the native component? Wouldn't that cause different behaviors in devices with different pixel densities?

Reply:
You can convert `10px` to `10` when covering to the RN transform array syntax